ParisBNF/abb/BNFabb152.xml

```diff
@@ -672,61 +681,70 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   chemin. En effet, deux textes édités et traduits par J. Kolmodin mentionnent Dāflā
                   et précisent son statut et celui de son voyage vers Jérusalem. Ainsi, selon les
                   « Annales du Père Māḥṣanta Māryām » en 1514 (année 168 de la Miséricorde), le
-                    <roleName type="other">kantibā</roleName> Dāflā, accompagné du fils de Amda Mikā’ēl, se
-                  rendent à Jérusalem « sur ordre du roi » (<bibl>Kolmodin, 1912, p. 25 (tx), p. 33
-                    (tr.) </bibl>). Puis, salon une liste de <term>tazkār</term> initiant la
-                  Chronique de la famille d’Asgadom, fils de Zar’āy, le <roleName type="other">kantibā</roleName>
+                    <roleName type="title">kantibā</roleName> Dāflā, accompagné du fils de Amda Mikā’ēl, se
+                               rendent à Jérusalem « sur ordre du roi » (<bibl><ptr target="bm:Kolmodin1912TraditionsT"/><citedRange>25</citedRange> (tx) </bibl>, <bibl><ptr target="bm:bm:Kolmodin1915TraditionsV"/><citedRange>33 (tr.)</citedRange></bibl>). Puis, salon une liste de <term>tazkār</term> initiant la
+                  Chronique de la famille d’Asgadom, fils de Zar’āy, le <roleName type="title">kantibā</roleName>
                   Dāflā meurt le 5 miyāzyā de l’an 183 de la Miséricorde, en mars 1531, probablement
-                  lors d’une des plus grandes victoires de l’<roleName type="other">imam</roleName> Ahmed, dans
-                  le Dawāro. (<bibl>Kolmodin, 1912, p. 53</bibl>).Éd. : CRLibAks II-38, p. 35 (txt),
-                  p. 41 (tr.) ; HuntLand 47. </note>
+                  lors d’une des plus grandes victoires de l’<roleName type="title">imam</roleName> Ahmed, dans
+                               le Dawāro. (<bibl><ptr target="bm:Kolmodin1912TraditionsT"/><citedRange unit="page">53</citedRange></bibl>).Éd. : <bibl><ptr target="bm:ContiRossini1909LiberT"/><citedRange>II-38,</citedRange><citedRange unit="page">35 (text)</citedRange></bibl>
+                               <bibl><ptr target="bm:ContiRossini1910LiberV"/><citedRange unit="page">41 (tr.)</citedRange></bibl> ; 
+                               <bibl> <ptr target="bm:Huntingford1965Land"/><citedRange unit="page">47</citedRange></bibl>. 
+                               <date type="production" when="1514" cert="medium">ca. 1514 (<date notAfter="1531" notBefore="1508" ></date>)</date>
```

Comment:
There is a bit of interpretation in this split, but if you are sure this what was meant ok.

Reply:
Yes I am sure, the text and translation are two separate items